CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 #### Assembly
 - Added new instructions: `is_odd`, `assert_eqw`, `mtree_merge`.
 - [BREAKING] Removed `mtree_cwm` instruction.
+- [BREAKING] Renamed `Read/ReadW` operations into `AdvPop/AdvPopW`.
```

Comment:
I think this could go into "VM internals" section as it doesn't directly change the assembly language.